<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWidget</class>
 <widget class="QDialog" name="ProfileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Профиль</string>
  </property>
  <widget class="QGroupBox" name="tagGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST type B</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Тег</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <widget class="QLabel" name="tagLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GOST type B</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tag</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="nameGroupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST type B</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Имя</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>310</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GOST type B</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="editNamePushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>18</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST type B</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="loginGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST type B</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Логин</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GOST type B</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="editPasswordPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>GOST type B</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сменить пароль</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
